scripts/model_gen_ng/model_gen_ng/templates/struct_late.tpl

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+[# --- YAML traits --- -#]
+INTROSPECTION_NS([[- struct.user_namespace ]], [[ struct.name ]],
+                [%- for field in struct.all_fields -%]
+                [[- field.name ]][% if not loop.last %], [% endif -%]
+                [%- endfor %])
```

Comment:
OK, I think it's now a good to time unwrap the macro castle in here.

Reply:
Done!